agent/core/src/test/java/org/jolokia/handler/ListHandlerTest.java

```diff
@@ -168,7 +168,7 @@ public void truncatedList() throws Exception {
         JmxListRequest request = new JmxRequestBuilder(RequestType.LIST).pathParts("java.lang", "type=Runtime").build();
         Map res = execute(request);
         assertFalse(res.containsKey("op"));
-        assertEquals(res.size(),2);
+        assertEquals(res.size(),3);
```

Comment:
and might need to update https://github.com/rhuss/jolokia/blob/b6010e3c053648eb280e57c1255f741d579b0688/src/docbkx/protocol/list.xml#L158 then, will do it tomorrow if your answer is positive